radio/src/gui/480x272/model_setup.cpp

```diff
@@ -972,7 +981,9 @@ bool menuModelSetup(event_t event)
         }
         break;
       }
-      case ITEM_MODEL_EXTERNAL_MODULE_OPTIONS: {
+
+      case ITEM_MODEL_EXTERNAL_MODULE_OPTIONS:
+      {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Extract this nested code block into a separate method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=cpp%3AS1199)